src/index.js

```diff
@@ -0,0 +1,42 @@
+const inquirer = require("inquirer");
+const utilities = require("./utilities");
```

Comment:
Would be nice to see the required function destructured from the object directly here:

` const { writeToFile } = require("./utilities")